### DISCUSSION TITLE
Responsive Button Width

### DISCUSSION THREAD

**Discussion by chess-server** [palcarazm/bootstrap5-toggle#115] [Q&A] [answered]
On standard bootstrap buttons, you can use btn-block and the button fills the parent element, mainly a div, automatically. When changing screen size or switching device, the button size changes accordingly.

How can I achieve this with the bootstrap5-toggle? 

Using the toggle in a <table> which should have equal width columns leads to unequal width when the toogle is too wide.

**Comment by palcarazm** [comment, marked as answer]
Can you please provide a code example?

**Reply by palcarazm** [reply]
Hi @chess-server,
what you are looking for is something like that https://codepen.io/palcarazm/pen/RwYrYQL ?

**Reply by chess-server** [reply]
Hi, yes, this is exactly what I am looking for. I am more  a backend developer, I was not aware of this way to access the parent element width. 

Maybe this is a useful extension of the documention!

**Reply by palcarazm** [reply]
I think I can correct directly this in the code to make all toggles responsive. I've to check if there are no render issues in some specifics cases.

**Reply by palcarazm** [reply]
It's fixed! In the next version, it will be included.
